providers/monitor.rb

```diff
@@ -10,11 +10,7 @@ def whyrun_supported?
 action :add do
   Chef::Log.debug "Adding monitoring for #{new_resource.name}"
   template ::File.join(yaml_dir, "#{new_resource.name}.yaml") do
-    if node['platform_family'] == 'windows'
-      owner 'Administrators'
-      rights :full_control, 'Administrators'
-      inherits false
-    else
+    unless node['platform_family'] == 'windows'
```

Comment:
what about Agent 5, should we also not set permissions here if Agent 5 is installed? :)

Reply:
It works this way but you're right that for consistency sake, I'll not change it for Agent 5 (it's unchanged for A5 in `dd-agent.rb`). Nice catch, thanks.